lib/run_argument_parser.py

```diff
@@ -231,3 +231,8 @@ def __init__(self):
             help="Comma-separated list of spacers to use for testing junction epitopes. Include None to test junctions without spacers. Peptide combinations will be tested with each spacer in the order specified.",
             default="None,AAY,HHHH,GGS,GPGPG,HHAA,AAL,HH,HHC,HHH,HHHD,HHL,HHHC"
         )
+        self.parser.add_argument(
+            '--max-clip-length', type=int,
+            help="Number of amino acids to permit clipping from the start and/or end of peptides in order to test novel junction epitopes when the first pass on the full peptide fails.",
+            default=3,
+        )
```

Comment:
My initial inclination was to limit the final peptide size, rather than limit the clipping length, but I think I like this better upon review.  It's intuitive and simplifies a lot of the code.

Reply:
I do have a commit locally to would use a minimum_peptide_length instead of the max_clip_length. Do you want me to push it so you can see how it would be different? Maybe in a separate PR?

Reply:
yeah, sure - I think either could work, and the min_peptide_length might be more intuititive ot people